src/DataViews/include/DataViews/SamplingReportDataView.h

```diff
@@ -63,11 +63,18 @@ class SamplingReportDataView : public DataView {
   // The callstack view will be updated according to the visible selected addresses and thread id.
   void UpdateVisibleSelectedAddressesAndTid(const std::vector<int>& visible_selected_indices);
 
+  // Returns the value to the right from ± in notation like `3 ± 0.2%`.
```

Comment:
Nit: you might want to add to the comment, that you are taking the max of the distances.

Reply:
Also consider having this (and the next function) as static/free function in the cpp file.

Reply:
How can they be free/static if their implementation relies on  `app_` and `thread_events_count_`. Or do you suggest to pass all the stuff as arguments?

Reply:
Well you can pass app_ and thread_events_count_ to them. But I guess it is fine.